docs/DeveloperGuide.md

```diff
@@ -446,7 +448,7 @@ The sequence diagram below shows what happens when a Description object is insta
   Returns a `Description` object containing the String with the description being "First Interaction".
 
 * `isValidDescription(String test)` <br>
-  Checks if the parsed String is a valid input. Returns true if the input String is alphanumeric and does not exceed 280 characters,
+  Checks if the parsed String is a valid input. Returns true if the input String is not blank and does not exceed 280 characters,
```

Comment:
Checks if the **given** String is a valid input. Returns true if the input `String` is not blank and does not exceed 280 characters,

or remove the first sentence